mystery_word.py

```diff
@@ -0,0 +1,103 @@
+import random
+import re
+
+with open("/usr/share/dict/words") as w:
+    word_list = w.readlines()
+
+guessed = []
+revealed = []
+
+
+def random_word_assignment():
+    global random_word
+    random_word = random.choice(word_list).strip().lower()
+    return random_word
+
+
+def game_intro(random_word):
+    print_statement = "The word has {} letters. You have 8 guesses.".format(
+        len(random_word), counter)
+    print(print_statement)
+    return print_statement
+
+
+def change_count_guess(counter):
+    counter -= 1
+    guessed.append(guess)
+    print(counter_statement)
+    return counter
+
+
+# def display_counter():
+#     print(counter_statement)
+
+
+def correct_guess(guess, revealed):
+    revealed.append(guess)
+    print("Good job! " + counter_statement)
+    return revealed
+
+
+def incorrect_guess(guess, counter):
+    counter -= 1
+    guessed.append(guess)
+    print("Nope. " + counter_statement)
+    return guessed
+
+
+def display_correct_guesses(guess):
+    print(" ".join(char if char in revealed else "_" for char in random_word))
+    return (" ".join(char if char in revealed else "_" for char
+            in random_word))
```

Comment:
Repeated code breaks so easily -- if you're going to do this, save the result in a variable and then print and return it.